perun-core/src/main/java/cz/metacentrum/perun/core/api/VosManager.java

```diff
@@ -46,7 +46,7 @@
 	 * @throws RelationExistsException
 	 * @throws InternalErrorException
 	 */
-	List<Vo> getVos(PerunSession perunSession) throws InternalErrorException, PrivilegeException;
+	List<Vo> getVos(PerunSession perunSession) throws InternalErrorException, PrivilegeException, PolicyNotExistsException;
```

Comment:
Is this exception still needed in Entry/Api?

Reply:
No, it is not. Fixed